repo.json

```diff
@@ -151,17 +151,8 @@
                         "o3de-sdk>=2.1.0",
                         "o3de>=2.1.0"
                     ],
-                    "dependencies": [
-                        "Atom_RPI",
-                        "Atom_Feature_Common",
-                        "Atom_Component_DebugCamera",
-                        "CommonFeaturesAtom",
-                        "PhysX",
-                        "PrimitiveAssets",
-                        "StartingPointInput"
-                    ],
                     "download_source_uri": "https://github.com/o3de/o3de-extras/releases/download/2.0/ros2-2.0.0-gem.zip",
-                    "sha256": "8910f69ffb7598a6c0880f44fb6785edfa90e7de47becdff4358c978be848fd4"
```

Comment:
After this change, we probably need to bump the ROS2 gem version and upload a new remote gem archive for the development branch because it looks like ROS2 version 2.0.0 is based on the content in the stab branch.